core/render/vi/__init__.py

```diff
@@ -105,7 +105,7 @@ def getVersion(*args, **kwargs):
     Returns viur-core version number
     """
     current.request.get().response.headers["Content-Type"] = "application/json"
-    if conf["viur.instance.is_dev_server"]:
+    if conf["viur.instance.is_dev_server"] or current.user.get():
```

Comment:
Hello, we had a discussion today, that the version string should generally be exposed fully. I would like to discuss this later in a separate pull request.